kyuubi-server/src/main/scala/org/apache/kyuubi/engine/ProcBuilder.scala

```diff
@@ -156,6 +155,8 @@ trait ProcBuilder {
 
   private val engineLogMaxLines = conf.get(KyuubiConf.SESSION_ENGINE_STARTUP_MAX_LOG_LINES)
 
+  private val engineDestroyTimeOut = conf.get(KyuubiConf.SESSION_ENGINE_DESTROY_TIMEOUT)
```

Comment:
```suggestion
  private val engineStartupDestroyTimeout = conf.get(KyuubiConf.SESSION_ENGINE_STARTUP_DESTROY_TIMEOUT)
```

Reply:
“process.waitFor( engineDestroyTimeOut , TimeUnit . MICROSECONDS )”   also needs to be modified